config/internal/apiserver/default/deployment.yaml.tmpl

```diff
@@ -96,6 +96,14 @@ spec:
               value: "{{.APIServer.CacheImage}}"
             - name: MOVERESULTS_IMAGE
               value: "{{.APIServer.MoveResultsImage}}"
+            {{ if .MLMD.Deploy }}
```

Comment:
for v2 mlmd is a hard requirement right, so the plan is to remove this at some point right?

Reply:
if it's a hard requirement, presumably that's a change to the CR behavior (MLMD would now default to true and/or be required)

Reply:
by having this here it gives us some flexibility for both v1 and v2